# this string allows to change the path to the Haar classifier model
# string
# data_directory: path_to_data -> please change this parameter in the launch file

# The factor by which the search window is scaled between the subsequent scans
# double
faces_increase_search_scale: 1.1

# Minimum number (minus 1) of neighbor rectangles that makes up an object.
# int
faces_drop_groups: 3

# Minimum search scale x
# int
faces_min_search_scale_x: 20

# Minimum search scale y
# int
faces_min_search_scale_y: 20

# when this flag is activated, the 3d size of a face is determined and only reasonable sizes may continue in the processing pipeline
# bool
reason_about_3dface_size: true

# the maximum feasible face diameter [m] if reason_about_3dface_size is enabled
# double
face_size_max_m: 0.35

# the minimum feasible face diameter [m] if reason_about_3dface_size is enabled
# double
face_size_min_m: 0.1

# maximum distance [m] of detected faces to the sensor
# double
max_face_z_m: 8.0

# enables some debug outputs
# bool
debug: false
